cairo/src/Slay/Cairo.hs

```diff
@@ -124,3 +124,11 @@ instance RenderElement g (PrimCurve g) where
     Cairo.moveTo (x' + x1) (y' + y1)
     curveThrough p1 p2 p3
     Cairo.stroke
+
+instance RenderElement g (PrimCircle g)   where
+  renderElement (Circle cc a b c) getG (Offset x y) = do
+    for_ (getG cc) $ \bc -> do
```

Comment:
Why make the color optional for a circle? It's optional for rectangles because sometimes rectangles can be used just to add padding (and therefore must be transparent). I don't think circles are useful for this.